Formula/sbom-tool.rb

```diff
@@ -0,0 +1,65 @@
+class SbomTool < Formula
+  desc "Scalable and enterprise ready tool to create SBOMs for any variety of artifacts"
+  homepage "https://github.com/microsoft/sbom-tool"
+  url "https://github.com/microsoft/sbom-tool/archive/refs/tags/v1.1.8.tar.gz"
+  sha256 "86653b76ef78ed2fb43b47fc6f3b38118f0799ebbc6d4ee92afa48ef12a997fd"
+  license "MIT"
+  head "https://github.com/microsoft/sbom-tool.git", branch: "main"
+
+  depends_on "dotnet" => :build
+  # currently does not support arm build
+  # upstream issue, https://github.com/microsoft/sbom-tool/issues/223
+  depends_on arch: :x86_64
+
+  uses_from_macos "icu4c"
```

Comment:
There doesn't appear to be linkage with any ICU libraries. Are you sure this is needed? Maybe it is needed only at build-time?

Reply:
Sounds about right, let's use it in the build time